Doc/whatsnew/3.7.rst

```diff
@@ -1012,90 +1777,213 @@ Other CPython Implementation Changes
   ``__loader__`` (previously, the former was set to ``None``).  See
   :issue:`32303`.
 
+* The :func:`locals` dictionary now displays in the lexical order that
+  variables were defined.  Previously, the order was reversed.
```

Comment:
Actually the order was arbitrary before 3.6 (as the order of all dict items). I would remove the note about the reversed order.